mizar/obj/data_networkpolicy.py

```diff
@@ -0,0 +1,61 @@
+# SPDX-License-Identifier: MIT
+# Copyright (c) 2020 The Authors.
+
```

Comment:
why not just name the file networkpolicy.py

Reply:
If it's networkpolicy.py, it's data structure for holding networkpolicy object. Here it's not. It's some sub structures under networkpolicy.

Reply:
That's fine then